### ISSUE TITLE
Your creadential have expired

### ISSUE THREAD

**Issue by MarieLePanda** [RickVanRousselt/SharePointAdminBot#15]
Hi, i would test your admin bot, I have run the bot on local, I can login and it's work. When I ask to get information on sp site, he ask me the ur, when I give him I have the following error
 
![image](https://user-images.githubusercontent.com/3236036/35146108-4aa1ce68-fd0a-11e7-983a-adbdd643a8a6.png)

**Comment by RickVanRousselt** [comment]
Hi,
You are running the bot locally. Did you fork the code and did you follow the [install guide](https://github.com/RickVanRousselt/SharePointAdminBot/wiki/How-to-Install)? Did you change the setting in your Azure AD app to multi-tenant?

**Comment by MarieLePanda** [comment]
Yes I fork the code, but I didn't follow your install guide, I run the code, like I'm use when I develop chatbot and graph api project. I will try to follow the guide when I have time and come back update my post.
